lmod_proxy/cmd.py

```diff
@@ -1,22 +1,17 @@
 # -*- coding: utf-8 -*-
 """Command line debug start of flask application"""
-import logging
 import os
 
-from lmod_proxy import web
-
-log = logging.getLogger('lmod_proxy')
+from lmod_proxy import config
 
 
 def run_server():
-    logging.basicConfig(level=logging.DEBUG)
+
```

Comment:
nit: Though it's not a requirement of pep8/pyflakes, want to add a docstring here?